pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1206,14 +1206,31 @@ func (t *Task) haveRsyncClientPodsCompletedOrFailed() (bool, bool, error) {
 					LastObservedProgressPercent: dvmp.Status.LastObservedProgressPercent,
 					LastObservedTransferRate:    dvmp.Status.LastObservedTransferRate,
 				})
+			case dvmp.Status.PodPhase == corev1.PodPending:
+				t.Owner.Status.PendingPods = append(t.Owner.Status.PendingPods, &migapi.PodProgress{
+					ObjectReference:             objRef,
+					LastObservedProgressPercent: dvmp.Status.LastObservedProgressPercent,
+					LastObservedTransferRate:    dvmp.Status.LastObservedTransferRate,
+				})
+				pendingMessage += dvmp.Status.LogMessage + "\n"
 			}
 		}
 	}
 
 	isCompleted := len(t.Owner.Status.SuccessfulPods)+len(t.Owner.Status.FailedPods) == len(t.Owner.Spec.PersistentVolumeClaims)
 	hasAnyFailed := len(t.Owner.Status.FailedPods) > 0
-
-	return isCompleted, hasAnyFailed, nil
+	isAnyPending := len(t.Owner.Status.PendingPods) > 0
+
+	if isAnyPending {
+		t.Owner.Status.SetCondition(migapi.Condition{
+			Type:     RsyncClientPodsPending,
+			Status:   migapi.True,
+			Reason:   "PodStuckInContainerCreating",
+			Category: migapi.Warn,
+			Message:  pendingMessage,
```

Comment:
I wonder if there is some amount of redundancy here?

`Reason: PodStuckInContainerCreating` message: `fmt.Sprintf("Pod %s/%s is stuck in ContainerCreating for more than 10 mins\n", pod.Namespace, pod.Name)` 

I expect just the list of name of the pods in the message or something like 9/15 pods are stuck, something more general, less repetitive

Reply:
a message with `Name/namespace` of each pod that is in the pending state is more user friendly I think